cmd/minikube/cmd/start_flags.go

```diff
@@ -611,6 +611,9 @@ func generateNewConfigFromFlags(cmd *cobra.Command, k8sVersion string, rtime str
 			klog.Info("auto-setting LocalStorageCapacityIsolation to false because using btrfs storage driver")
 			cc.KubernetesConfig.FeatureGates = addFeatureGate(cc.KubernetesConfig.FeatureGates, "LocalStorageCapacityIsolation=false")
 		}
+		if runtime.GOOS == "linux" && si.DockerOS == "Docker Desktop" {
+			out.WarningT("For an improved experience it's recommended to use Docker Engine over Docker Desktop")
```

Comment:
instead of ...

also consider adding a todo: follow up add page to site

Reply:
Update message to include a link to Docker Engine install